<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormKurir</class>
 <widget class="QWidget" name="FormKurir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Kurir</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblJudul">
     <property name="styleSheet">
      <string notr="true">font-size:18px; font-weight:bold; color:#2b5fab;</string>
     </property>
     <property name="text">
      <string>Form Input Kurir</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>

   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblKurirID">
       <property name="text">
        <string>ID Kurir</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtKurirID"/>
     </item>

     <item row="1" column="0">
      <widget class="QLabel" name="lblNamaKurir">
       <property name="text">
        <string>Nama Kurir</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtNamaKurir"/>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="lblNoHP">
       <property name="text">
        <string>No. HP</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtNoHP"/>
     </item>

     <item row="3" column="0">
      <widget class="QLabel" name="lblEmail">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtEmail"/>
     </item>

     <item row="4" column="0">
      <widget class="QLabel" name="lblPlatNomor">
       <property name="text">
        <string>Plat Nomor</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtPlatNomor"/>
     </item>

     <item row="5" column="0">
      <widget class="QLabel" name="lblStatus">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cmbStatus">
       <item>
        <property name="text">
         <string>aktif</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nonaktif</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>

   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnSimpan">
       <property name="text">
        <string>Simpan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUbah">
       <property name="text">
        <string>Ubah</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCari">
       <property name="text">
        <string>Cari</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHapus">
       <property name="text">
        <string>Hapus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBersih">
       <property name="text">
        <string>Bersih</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <item>
    <widget class="QTableWidget" name="tableKurir">
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>ID Kurir</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nama Kurir</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>No. HP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Plat Nomor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
